common/src/tx_initialize_kernel_enter.c

```diff
@@ -133,6 +137,9 @@ VOID  _tx_initialize_kernel_enter(VOID)
        later used to represent interrupt nesting.  */
     _tx_thread_system_state =  TX_INITIALIZE_IN_PROGRESS;
 
+    /* Optional random number generator initialization.  */
+    TX_INITIALIZE_RANDOM_GENERATOR_INITIALIZATION
```

Comment:
Could you explain why we add this feature? and could we let user initialize the random number generator in application code before calling _tx_initialize kernel_enter()?